Firestore/Source/API/FIRDocumentSnapshot.mm

```diff
@@ -299,6 +250,6 @@ - (instancetype)initWithFirestore:(FIRFirestore *)firestore
   return data;
 }
 
-@end
+end
```

Comment:
Revert this change? (It should be `@end`).

Reply:
Done.